.ci/pipeline_definitions

```diff
@@ -1,24 +1,12 @@
 gardener-website-generator:
-  template: 'default'
   base_definition:
-    traits:
-      version: ~
```

Comment:
removing version handling seems fine as there are no release pipelines at all, so I assume no versioning (apart from git-based versioning) is required for this component

you can also omit VERSION file from repository